### PR TITLE
Add Security Considerations section on Protected Information.

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to address issue #215 by explaining what information is protected by a data integrity proof.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/271.html" title="Last updated on Jun 27, 2024, 9:59 PM UTC (b8690a7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/271/8afb5a8...b8690a7.html" title="Last updated on Jun 27, 2024, 9:59 PM UTC (b8690a7)">Diff</a>

### CODE REVIEW

**Review by dlongley — APPROVED** [review]

**Review by longpd — APPROVED** [review]

**Review by David-Chadwick — APPROVED** [review]

**Review by Wind4Greg — APPROVED** [review]

**Review by iherman — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -3932,6 +3932,52 @@ <h3>Transformations</h3>
...
       </section>
 
+      <section>
+        <h3>Protected Information</h3>
+
+        <p>
+The data that is protected by any [=data integrity proof=] is the
+[=transformation|transformed data=]. [=transformation|Transformed data=]
+is generated by a [=transformation algorithm=] that is specified by a
+particular [=cryptosuite=]. This protection mechanism is in contrast to some
```
Comment (TallTed) [inline index.html]:
```suggestion
particular [=cryptosuite=]. This protection mechanism differs from some
```

File: index.html [hunk 2]
```diff
@@ -3932,6 +3932,52 @@ <h3>Transformations</h3>
...
+        <p>
+Other [=cryptosuites=] such as
+<a data-cite="?VC-DI-ECDSA#ecdsa-rdfc-2019">ecdsa-rdfc-2019</a> and
+<a data-cite="?VC-DI-EDDSA#eddsa-rdfc-2022">eddsa-rdfc-2022</a> use
+[[[?RDF-CANON]]] to [=transformation|transform=] the data to canonicalized
+N-Quads [[?N-QUADS]], which is then cryptographically hashed and digitally
+signed. One benefit of this approach is that the cryptographic signature is
+portable to a variety of different syntaxes, such as JSON, YAML, and CBOR,
+without invalidating the digital signature. More traditional digital signature
+mechanisms do not have this capability.
```
Comment (TallTed) [inline index.html]:
```suggestion
signed. One benefit of this approach is that the cryptographic signature is
portable to a variety of different syntaxes, such as JSON, YAML, and CBOR,
without invalidating the signature. More traditional cryptographic signature
mechanisms do not have this capability.
```

File: index.html [hunk 3]
```diff
@@ -3932,6 +3932,52 @@ <h3>Transformations</h3>
...
+        </p>
+
+        <p>
+Implementers and developers are urged to not presume or trust information
+that contains a [=data integrity proof=] unless the proof has been [=verified=]
+and the verified data is provided in a return value from a software library
+that has confirmed that all data returned has been successfully protected.
```
Comment (TallTed) [inline index.html]: There might instead be some presumption which was inadvertently omitted from "presume ... information", and which restoration would make this sentence make more sense.
```suggestion
Implementers and developers are urged to not trust information
that contains a [=data integrity proof=] unless the proof has been [=verified=]
and the verified data is provided in a return value from a software library
that has confirmed that all data returned has been successfully protected.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**